test/gui/tst_syncing/test.feature

```diff
@@ -257,6 +257,20 @@ Feature: Syncing files
         Then the file "Folder1/a\\a.txt" should be blacklisted
 
 
+    Scenario: Filenames that are rejected by the server are reported
+        Given user "Alice" has set up a client with default settings
+        And user "Alice" has created a folder "Folder1" inside the sync folder
+        When the user waits for folder "Folder1" to be synced
+        And user "Alice" creates a file "Folder1/a\\a.txt" with the following content inside the sync folder
+            """
+            test content
+            """
+        Then as "Alice" folder "Folder1" should exist on the server
+        When the user clicks on the activity tab
+        And the user selects "Not Synced" tab in the activity
+        Then at least a file should be blacklisted
+
+
```

Comment:
There already us a scenario `Filenames that are rejected by the server are reported` which was skipped few months ago. It is better to fix that scenario instead of creating a new scenario. IMO, the fixed scenario's step would be something like this:
```
    Scenario: Filenames that are rejected by the server are reported
        Given user "Alice" has set up a client with default settings
        And user "Alice" has created a folder "Folder1" inside the sync folder
        When the user waits for folder "Folder1" to be synced
        And user "Alice" creates a file "Folder1/a\\a.txt" with the following content inside the sync folder
            """
            test content
            """
        Then as "Alice" folder "Folder1" should exist on the server
        When the user clicks on the activity tab
        And the user selects "Not Synced" tab in the activity
        And the user waits until at least a file is blacklisted
        Then the file "Folder1/a\\a.txt" should be blacklisted
```